go.mod

```diff
@@ -303,6 +303,9 @@ replace (
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.4
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.16.0
 
+	// Avoid CVE-2024-45337
+	golang.org/x/crypto => golang.org/x/crypto v0.31.0
+
```

Comment:
Not sure if we want to resolve all the indirect references to old versions of x/crypto lib. By doing so the scope of change becomes much bigger.

In master branch, currently only the direct vulneralbility is fixed by upgrading x/crypto to 0.32.0, without touching any of the indirect ones.